packages/lesswrong/components/sequences/SequencesGridItem.jsx

```diff
@@ -83,6 +78,7 @@ const styles = theme => ({
   },
 
   bottom: {
+    pointerEvents: "none",
```

Comment:
Why is this necessary? (I recall you talking about it with Oli and him being skeptical of pointer-event settings. I'm less skeptical of pointer event settings but also unsure why it's needed here, given that it wasn't needed in any of the other LinkCard components)

Reply:
Removed. It wasn't actually necessary; it was the remnant of a previous way of doing this that I tried, which wasn't the best way. (It also wasn't harmful, in this case, since nothing inside `classes.bottom` needed to be clickable.)